components/dashboard/script.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+
+const vars = {
+    REACT_APP_HOSTNAME : "Gitpod",
+    REACT_APP_DOCS_URL : "https://www.gitpod.io/docs/",
+    REACT_APP_SOCIAL_URL : "https://community.gitpod.io/",
+    REACT_APP_FAVICON_192 : "icons/favicon192.png",
+    REACT_APP_FAVICON_256 : "icons/favicon256.png",
```

Comment:
Since we have the mapping here. Do we really need to refactor paths ? eg icons/gitpod32.svg why can't they be as-is old names and paths

Reply:
Just better design. Placing all the static images and icons at a single place makes things clear

Reply:
We want to keep the structure as close as possible. No refactoring

Reply:
Then everything just looks messy. Some icons here some there, its not a good design

Reply:
We want to contribute, Why not contribute things which are simple to work around

Reply:
I agree with Pavan on this point. Refactoring and evaluating their design decisions would be a different chapter altogether.